src/components/Dropdown/Dropdown.st.css

```diff
@@ -27,6 +27,11 @@
 :import {
     -st-from: "./DropdownBase.st.css";
     -st-default: DropdownBase;
+}
+
+:import {
+    -st-from: "./DropdownNativeSelect.st.css";
+    -st-default: DropdownNativeSelect;
     -st-named: borderHoverColor;
```

Comment:
Is there a reason why `borderHoverColor` should be implemented in `DropdownNativeSelect` as well as the `DropdownBase`?

Reply:
actually - no 😂

Reply:
So maybe we can move it back 🤔 ?